src/osal/osal_freertos.h

```diff
@@ -145,7 +162,11 @@ typedef QueueHandle_t osal_queue_t;
 
 TU_ATTR_ALWAYS_INLINE static inline osal_queue_t osal_queue_create(osal_queue_def_t* qdef)
 {
-  return xQueueCreateStatic(qdef->depth, qdef->item_sz, (uint8_t*) qdef->buf, &qdef->sq);
+#if configSUPPORT_STATIC_ALLOCATION == 1
+	return xQueueCreateStatic(qdef->depth, qdef->item_sz, (uint8_t*) qdef->buf, &qdef->sq);
```

Comment:
Hi @hathach ,

I did check this and it seems it should always be available the way FreeRTOS is written, but for good measure I have added the ```#if configSUPPORT_STATIC_ALLOCATION``` clause to the ```sq``` variable, just in case anything changes in the future. 🙂 

Cheers,

Pete